core/new-gui/src/app/workspace/component/code-editor-dialog/code-editor-dialog.component.ts

```diff
@@ -57,6 +57,16 @@ export class CodeEditorDialogComponent implements AfterViewInit, SafeStyle, OnDe
   }
 
   ngOnDestroy(): void {
+    const dialog = document.getElementById("mat-dialog-udf");
+    if (dialog !== null) {
+      localStorage.setItem("udfDialogStyle", dialog.style.cssText);
```

Comment:
The life cycle uses localStorage, which can be discussed in the future. We can use it for now.